components/atom/input/src/Password/index.js

```diff
@@ -38,18 +34,21 @@ const Password = forwardRef(
     }
 
     return (
-      <div className={BASE_CLASS_PASSWORD}>
+      <>
         <Input
           ref={forwardedRef}
           {...props}
           onChange={handleChange}
           value={innerValue}
           type={type}
         />
-        <div onClick={toggle} className={BASE_CLASS_PASSWORD_TOGGLE_BUTTON}>
+        <span
+          onClick={toggle}
+          className={cx(BASE_CLASS_PASSWORD_TOGGLE_BUTTON)}
+        >
```

Comment:
Should this span have a `role=button` for accessibility purpose since is a toggle button?

Reply:
Retrocompatibility. Its a deprecated usage. Better do not modify it